build/bootstrap/bootstrap-debian.sh

```diff
@@ -55,9 +51,15 @@ sha256sum -c - <<EOF
 EOF
 sudo tar -C /usr/local -zxf /tmp/go.tgz && rm /tmp/go.tgz
 
-# Clone CockroachDB.
-git clone https://github.com/cockroachdb/cockroach "$(go env GOPATH)/src/github.com/cockroachdb/cockroach"
-git -C "$(go env GOPATH)/src/github.com/cockroachdb/cockroach" submodule update --init
+# Install Docker compose.
+sudo curl -L "https://github.com/docker/compose/releases/download/v2.31.0/docker-compose-linux-x86_64" -o /usr/local/bin/docker-compose
+sudo chmod +x /usr/local/bin/docker-compose
+docker-compose --version
+
+# Install NVM.
+# Note: you still required to run `nvm install <version>` to install a specific version of Node.js.
```

Comment:
We already install `node` above (line 29). Why is this necessary?

Reply:
If user need to develop code using a different version of Node, nvm is really useful in switching node version. Although L29 installation of `node` can be removed.

Reply:
Did you check what version of `node` is installed and whether you would want a different version?

Reply:
The current node version is `v16.20.2`. And managed-service I beleive [needs](https://github.com/cockroachlabs/managed-service/blob/master/console/ui/cc-console/package.json#L184C15-L184C22) `20.10.0` or above. Hence `nvm` would be useful for someone developing managed-service ui.